Essentials/src/plugin.yml

```diff
@@ -208,6 +208,10 @@ commands:
     description: Searches for an item.
     usage: /<command> <item>
     aliases: [dura,edura,durability,edurability,eitemdb,itemno,eitemno]
+  itemname:
+    description: Names an item.
+    usage: /<command> <name...>
```

Comment:
In line with other commands like `broadcast` and `ban`, this should be `<name>` without ellipsis.

Reply:
Completed as requested.